app/assets/config/manifest.js

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
shouldn't this be
```
//= link_tree ../images
//= link_directory ../javascripts .js
//= link_directory ../stylesheets .css
```
?

Reply:
That could be, I don't really understand the motives for this file. So I just followed a comment readable on the [link](https://github.com/rails/sprockets-rails/issues/444) on first description item, which created it that way. 
I'll do this change.